data_steward/analytics/cdr_ops/report_runner.py

```diff
@@ -265,7 +265,7 @@ def __call__(self, value, **kwargs):
 
 
 if __name__ == '__main__':
-    pipeline_logging.configure(logging.INFO, add_console_handler=True)
+   # pipeline_logging.configure(logging.INFO, add_console_handler=True) #q2
```

Comment:
I know this eliminated an error for you, but can you revert these changes to the report_runner? You can always comment it out for your local runs.

Reply:
i see. ok. changed back.